src/Utils/Markup_Utils.php

```diff
@@ -44,6 +44,23 @@ public static function class_attribute( $classes, $attribute = true ) {
 		);
 	}
 
+	/**
+	 * @param array $default
+	 * @param array $custom
+	 * @param bool  $to_string
+	 *
+	 * @return array|string
+	 */
+	public static function merge_classes( array $default, array $custom, bool $to_string ) {
+		$classes = ! empty( $custom ) ? array_merge( $default, $custom ) : $default;
+
+		if ( $to_string ) {
+			$classes = implode( ' ', $classes );
```

Comment:
Should this delegate to `self::class_attribute( $classes, false )` to take advantage of the added sanitization it brings?

How is this function different than `array_merge()`, aside from the third param? I.e., why wouldn't we call `Markup_Utils::class_attribute( array_merge( $default, $custom ), false )` to do the same thing?

Reply:
`Markup_Utils::class_attribute( array_merge( $default, $custom ), false )` would return a string correct? I think the ability to return an array is what's desired but I can get @ryanurban chime in.

Reply:
If you want two merge two arrays, use `array_merge( $default, $custom )`. There's no reason for us to wrap a standard PHP function.

Reply:
I did wonder if this should harness our `class_attribute` as well. Honestly I'm not sure why we'd need the ability to return a string, especially given what we do already with `class_attribute` or even `concat_attrs`. All I really care about is the ability to optionally specify core required classes and then add additional at will. Do with that what you will.

Reply:
@ryanurban : can you give an example of where you might use this, and why `array_merge()` wouldn't be just as useful in that circumstance?